cert-manager/cert-manager-webhook-role-binding/base/role-binding.yaml

```diff
@@ -0,0 +1,22 @@
+---
+
+# apiserver gets the ability to read authentication. This allows it to
+# read the specific configmap that has the requestheader-* entries to
+# api agg
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: cert-manager-webhook:webhook-authentication-reader
+  labels:
+    app: webhook
+    app.kubernetes.io/name: webhook
```

Comment:
these 2 labels can be removed since we'll add 
overlays/application to 
cert-manager                      
cert-manager-crds                 
cert-manager-webhook-role-binding

Can be a follow-up PR but needed for upgrade and delete.

Reply:
I'll remove them here and raise a follow-up PR for application.

Reply:
@kkasravi Looks like I might have to add the Application changes within this PR itself since these labes (especially the `app.kubernetes.io/name` labels) are spread across:
- cert-manager
- ca-injector
- webhook

I think we can break this down into 3 different folders representing these applications and then have an Application overlay for each, so that the right labels are applied. Folders would be: 
- cert-manager
- ca-injector
- webhook